### ISSUE TITLE
Can't import types with Typescript

### ISSUE THREAD

**Issue by MarcusElevait** [Ontotext-AD/graphdb.js#163]
**Library version [e.g. 1.1.2]**
2.0.2

**Node.js version [e.g. 8.16.0]**
16.14.0

**Describe the bug**
When trying to use the types like ServerClient, i cannot import it with typescript.

**To Reproduce**
I've added the graphdb lib with 'npm i graphdb'.
Then i added code for a simple connection to a graphdb:

```Typescript
const serverConfig = new ServerClientConfig('sampleurl/')
            .setTimeout(5000)
            .setHeaders({
                Accept: RDFMimeType.SPARQL_RESULTS_JSON,
            })
            .setKeepAlive(true);
const server = new ServerClient(serverConfig);
```

But no matter how i try to import ServerClient or ServerClientConfig, it cannot be found.

**Expected behavior**
I expect to easily import the types to use it in the code.

**Comment by svilenvelikov** [comment]
Hello [MarcusElevait](https://github.com/MarcusElevait)
Here is an example of how you can import types from the library. It's not mandatory to use nestjs, I just used it for faster proof that it works. There are two ways you can import from the library: first is by using commonjs style impot with `require` (see the commented out lines) and the other is with the standard es6 `import` declarations.

```javascript
import { Controller } from '@nestjs/common';
// const {ServerClient, ServerClientConfig} = require('graphdb').server;
// const {RDFMimeType} = require('graphdb').http;
import ServerClientConfig from 'graphdb';
import ServerClient from 'graphdb';
import RDFMimeType from 'graphdb';

@Controller()
export class TestController {
    test(): void {
        const serverConfig = new ServerClientConfig('sampleurl/')
            .setTimeout(5000)
            .setHeaders({
                Accept: RDFMimeType.SPARQL_RESULTS_JSON,
            })
            .setKeepAlive(true);
            const server = new ServerClient(serverConfig);
    }
}
```

Please give us more info about your project configuration and the exact errors you receive if possible if you still have throubles.

**Comment by MarcusElevait** [comment]
Hi @svilenvelikov,
thanks for the fast answer.
Unfortunately this is not working for us. We also use nestjs and i set it exactly as you did and then at runtime i get the error: `graphdb__WEBPACK_IMPORTED_MODULE_8___default(...) is not a constructor`

We now managed to do it with:
```Typescript 
import { server as graphdbServer, http as graphdbHttp } from 'graphdb';
```
But then we have no possibility to directly jump to the source in the IDE.

But interesting that it is working with your setup.
We are using Typescript 4.6.3.
This is our tsconfig.json
```json
{
    "compileOnSave": false,
    "compilerOptions": {
        "rootDir": ".",
        "sourceMap": true,
        "declaration": false,
        "moduleResolution": "node",
        "emitDecoratorMetadata": true,
        "experimentalDecorators": true,
        "importHelpers": true,
        "target": "ES5",
        "module": "esnext",
        "typeRoots": ["node_modules/@types"],
        "lib": ["es2019", "dom"],
        "skipLibCheck": true,
        "skipDefaultLibCheck": true,
        "baseUrl": ".",
        "resolveJsonModule": true,
        "types": ["node", "jest"],
        "emitDecoratorMetadata": true,
        "target": "ESNext"
    },
    "exclude": ["node_modules", "tmp"]
}
```

Do you see something that is possibly wrong/missing in our setup?
Do you need some more information?

**Comment by svilenvelikov** [comment]
Indeed it breaks with 
```
[Nest] 1752  - 06/13/2022, 3:40:13 PM   ERROR [ExceptionsHandler] graphdb_1.default is not a constructor
TypeError: graphdb_1.default is not a constructor
```
when using `import`. With commonjs style it works though. 
We will investigate further what might be the reason for this.

**Comment by brownjasonj** [comment]
any further updates on this?  I have the same problem as I am using ESM and not commonjs modules so the above just doesn't work.  Is it possible to convert to using ESM?

**Comment by svilenvelikov** [comment]
This was fixed in [v3.0.1](https://github.com/Ontotext-AD/graphdb.js/releases/tag/v3.0.1)
